tests/index.test.js

```diff
@@ -180,22 +179,30 @@ describe('has one package and a delivery', () => {
 })
 
 describe('has two packages with different shipping estimates', () => {
-  it('should create one parcel with the latest estimate', () => {
-    const items = createItems(2)
-    const packageAttachment = {
-      packages: [
-        createPackage([
-          { itemIndex: 0, quantity: 1 },
-          { itemIndex: 1, quantity: 1 },
-        ]),
-      ],
-    }
-    const selectedAddresses = [residentialAddress]
+  const items = createItems(2)
```

Comment:
Is this inside an `it` statement? Im confused with git diff

Reply:
It is not. I've moved it to the `describe` scope to reuse the values on the other test. Do you prefer that I keep it inside the `it`scope?

Reply:
@kaisermann Yeah, I think is better to that all variables live on the `it` scope, so the tests can modify their dependencies easier if needed.

But it doesn't mean you cant create `const` values outside all `describe` context, like:
```js
const BASE_ITEMS = createItems(2)
const BASE_PACKAGE_ATTACHMENT = {
      packages: [
        createPackage([
          { itemIndex: 0, quantity: 1 },
          { itemIndex: 1, quantity: 1 },
        ]),
      ],
    }
    const SELECTED_ADDRESSES = [residentialAddress]
```

And on the `it` scope you do:
```js
const items = BASE_ITEMS
const packageAttachment = BASE_PACKAGE_ATTACHMENT
const selectedAddresses = SELECTED_ADDRESSES
```

What you think?

Reply:
The values vary a lot on other tests. I agree with you but I don't think it fits this scenario. I've just commited with the consts insided each `it`